integration-tests/test_redis_to_cassandra.py

```diff
@@ -10,7 +12,9 @@
 TABLE = f"{KEYSPACE}.emp"
 
 
-def test_redis_to_cassandra():
+@pytest.fixture(scope="module")
+def prepare_db(request):
```

Comment:
can we omit `request`? if not, can you add its type?

Reply:
removed